comparators/motorway_trunk_geometry_changed.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var deepEquals = require('deep-equals');
+var MAJOR_ROAD_TYPES = ['motorway', 'trunk', 'motorway_link', 'trunk_link'];
+
+function getHighwayType(feature) {
+  return feature.properties.highway;
+}
+
+function isMajorRoad(feature) {
+  var highwayType = getHighwayType(feature);
+  return MAJOR_ROAD_TYPES.indexOf(highwayType) !== -1;
+}
+
+function getGeometry(feature) {
+  return feature.geometry;
+}
+
+function geometryChanged(newVersion, oldVersion) {
+  if (!oldVersion && newVersion.deleted) {
+    return false;
+  }
+
+  if (!oldVersion && !newVersion.deleted) {
+    return false;
+  }
```

Comment:
👋 , can you explain when you would like to run this compare function, feature addition/modification/deletion ?

Reply:
only for geometry changes @amishas157

Reply:
Yeah that line read odd for me as well @amishas157 going to change it